daml-foundations/daml-tools/daml-cli/DA/Cli/Visual.hs

```diff
@@ -74,60 +78,98 @@ darToWorld manifest pkg = AST.initWorldSelf pkgs pkg
     where
         pkgs = map dalfBytesToPakage (dalfsContent manifest)
 
+tplName :: LF.Template -> String
+tplName LF.Template {..} = T.unpack $ head (LF.unTypeConName tplTypeCon)
```

Comment:
We should not `unpack` here but rather return a `Text`. This will make code below simpler as well.

Reply:
Cool thanks that simplified few things